common/workunit/package.cpp

```diff
@@ -17,6 +17,7 @@
 
 
 #include "platform.h"
+#include "jlib.hpp"
```

Comment:
I'm guessing it was because anything that includes jsuperhash.hpp or jhash.hpp will require jlib.hpp to have been included in order to compile. But the correct fix for that would be to have those include files include jlib.hpp themselves, I would think.